tutorials/large-messages/index.mdx

```diff
@@ -0,0 +1,377 @@
+---
+meta:
+  title: Create a serverless architecture that manage large messages, with Scaleway Messaging and Queuing NATS, Serverless Functions and Object Storage.
+  description: This page show an example of serverless architecture for asynchronous files conversion based on messages, functions, and triggers.
+content:
+  h1: Create a serverless architecture that manage large messages, with Scaleway Messaging and Queuing NATS, Serverless Functions and Object Storage.
+  paragraph: This page show an example of serverless architecture for asynchronous files conversion based on messages, functions, and triggers.
+categories:
+  - messaging
+  - functions
+tags: message nats serverless serverless-functions functions serverless-triggers triggers storage object-storage terraform large-messages
+dates:
+  validation: 2024-01-22
+  posted: 2024-01-22
+---
+
+# Introduction
+
+<Macro id="iam-requirements" />
+ 
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [set up authentication for the Terraform provider](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs#authentication)
+</Message>
+
+In this tutorial, we will see how to create a simple architecture that stores images and converts them to PDF automatically. 
+The goal is to understand how to transfer large message through a messaging service. 
+To do this, we will use a NATS server available in the Scaleway messaging service. 
+We will also use the Object storage service to store our images and the Serverless Function service to convert them.
+You can retrieve the full source code in [this repository](https://github.com/rouche-q/serverless-examples/tree/main/projects/large-messages/README.md)
+
```

Comment:
One other thing it would be nice to include in the intro:

We show how to provision all the required Scaleway resources via Terraform, but you can also use the [console](https://console.scaleway.com/), the [API](https://www.scaleway.com/en/developers/api/) or one of our other supported [developer tools](https://www.scaleway.com/en/developers/).

Reply:
I agree, done